server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -793,6 +818,26 @@ void handleSimpleQuery(ByteBuf buffer, final DelayableWriteChannel channel) {
         }
     }
 
+    private void handleCancelRequestBody(ByteBuf buffer, Channel channel) {
+        var keyData = readCancelRequestBody(buffer);
+        Session s = activeSessions.get(keyData); // the session executing the target query
+        if (s != null) {
+            String userName = s.sessionContext().sessionUser().name();
+            if (s.getMostRecentJobID() != null) {
+                transportKillJobsNodeAction.broadcast(
+                    new KillJobsRequest(List.of(s.getMostRecentJobID()),
+                                        userName,
+                                        "Cancellation requested by: " + userName));
+            } else {
+                LOGGER.trace("Cancellation request is ignored since the session does not have an active job to kill");
```

Comment:
```suggestion
                LOGGER.debug("Cancellation request is ignored since the session does not have an active job to kill");
```

I think these don't happen too frequently so may be possible to log them on the debug level instead of trace